style.css

```diff
@@ -1,9 +1,24 @@
-.wrapper {
+.home-wrap {
   max-height: 100vh;
-  min-width: 500px;
   background: url(images/Rectangle\ 5.png) 32% 0% no-repeat, url(images/Ellipse\ 3.png) 24% 85% no-repeat, url(images/Rectangle\ 4.png) 32% 23% no-repeat, url(images/Rectangle\ 5.png) 67% 39% no-repeat, url(images/Ellipse\ 15.png) 32% 49% no-repeat, url(images/Ellipse\ 5.png) 37% 67% no-repeat, url(images/Ellipse\ 5.png) 45% 62% no-repeat, url(images/Ellipse\ 13.png) 37% 44% no-repeat, url(images/Ellipse\ 12.png) 69% 15% no-repeat;
 }
 
+.home-wrap > div {
+  min-height: 500px;
+}
+
+.work-section {
+  display: inline-grid;
+  grid-template-columns: 100%;
+  row-gap: 20px;
+  column-gap: 20px;
```

Comment:
- [ ] [REQUIRED] You did a very good job on the design in the projects section but your design is not as required. The main container (`.work-section`) of the projects should be a `display:grid`.

- [ ] [OPTIONAL] Please select the correct color and font thickness for the project card 

![image](https://user-images.githubusercontent.com/63915024/170584394-8d17b8e1-e734-48e0-9578-7649b47f7b59.png)

Reply:
I just set it as  display: grid;

Reply:
done